shell/platform/android/external_view_embedder/external_view_embedder.cc

```diff
@@ -73,6 +75,16 @@ void AndroidExternalViewEmbedder::SubmitFrame(
     return;
   }
 
+  auto submit_info = frame->submit_info();
+  int64_t vsync_id = submit_info.vsync_id;
+  if (vsync_id) {
+    // Steal the vsync_id so that the frame submission doesn't use it
```

Comment:
Why don't we want the frame submission to use it?

Reply:
We need to wrap all updates within `AndroidSurfaceTransaction::Begin()` and `AndroidSurfaceTransaction::End()`. If vsync_id is passed to the frame, `AndroidSurfaceGLSkia` will do that wrapping. We need that not to happen so our transaction will also contain the updates to platform views.

Reply:
Can you please add a comment here to explain that?

Reply:
Added a comment